RELEASING.md

```diff
@@ -0,0 +1,63 @@
+# Release Checklist
+
+These steps assume that you've checked out the Soketto repository and are in the root directory of it.
+
+1.  Ensure that everything you'd like to see released is on the `develop` branch.
+
+2.  Create a release branch off `develop`, for example `release-v0.6.0`. The branch name should start with `release`
+    so that we can target commits with CI. Decide how far the version needs to be bumped based on the changes to date.
+    If unsure what to bump the version to (e.g. is it a major, minor or patch release), check with the Parity Tools team.
+
+3.  Check that you're happy with the current documentation.
+
+    ```
+    cargo doc --open
+    ```
+
+    CI checks for broken internal links at the moment. Optionally you can also confirm that any external links
+    are still valid like so:
+
+    ```
+    cargo install cargo-deadlinks
+    cargo deadlinks --check-http
+    ```
+
+    If there are minor issues with the documentation, they can be fixed in the release branch.
+
+4.  Bump the crate version in `Cargo.toml` to whatever was decided in step 2.
+
+5.  Update `CHANGELOG.md` to reflect the difference between this release and last. If you're unsure of
+    what to add, check with the Tools team.
+
+    One way to gain some inspiration on what to write is to look at the [closed PRs](https://github.com/paritytech/soketto/pulls?q=is%3Apr+is%3Aclosed).
+
+    You can also look through the commit history to find the code changes since the last release (eg `git log --pretty LAST_VERSION_TAG..HEAD`).
+
+6.  Commit any of the above changes to the release branch and open a PR in GitHub with a base of `master`.
+
+7.  Once the branch has been reviewed and passes CI, it can be merged to `master`.
```

Comment:
It should be merged to **both** master and develop right?

Reply:
Yup; for now I put the "merge back to develop" at the end, and from `master` (I geuss just incase we end up having issues publishign on master and need to change anything else)